.pylintrc

```diff
@@ -1,3 +1,6 @@
+[MASTER]
+init-hook="from pylint.config import find_pylintrc; import sys; from os import path; sys.path.append(path.join(path.dirname(find_pylintrc()),'./src'))"
+
```

Comment:
This looks like a really hacky solution :thinking: 
if I get it correctly you've been trying to fix pylint errors that appears at [0a2d679](https://github.com/iScsc/RootPythia/pull/9/commits/0a2d679d4c1381b807994a5d0beb3a0034d666c0) (with this [check fail report](https://github.com/iScsc/RootPythia/pull/9/checks?check_run_id=13280838590)). 
However this fail was introduced by [f7a764f](https://github.com/iScsc/RootPythia/pull/9/commits/f7a764f2b384b2ab2e7bc11bded617e781bcd6a0) because recreating the `__init__.py` (which is needed for pytest test units) make src a python module, you then have to import from `src.`! (I guess) 

I've reproduced it locally see:
```
 $ git show
commit 0a2d679d4c1381b807994a5d0beb3a0034d666c0 (HEAD)
Author: xlitoni <vacherquentin@outlook.fr>
Date:   Sat May 6 09:29:51 2023 +0200

    go back to realpath
   ....
 $ pylint --rcfile=.pylintrc $(git ls-files '*.py')
************* Module src.main
src/main.py:4:0: E0401: Unable to import 'classes.challenge' (import-error)
src/main.py:5:0: E0401: Unable to import 'classes.user' (import-error)
src/main.py:6:0: E0401: Unable to import 'pngmaker.new_validated_challenge' (import-error)
************* Module new_validated_challenge
src/pngmaker/new_validated_challenge.py:3:0: E0401: Unable to import 'classes.challenge' (import-error)
src/pngmaker/new_validated_challenge.py:4:0: E0401: Unable to import 'classes.user' (import-error)

-------------------------------------------------------------------
Your code has been rated at 7.62/10
```
The solution is then to prepend `src.` to your import paths:
```
 $ git diff
diff --git a/src/main.py b/src/main.py
index 4dc617c..a14d0be 100755
--- a/src/main.py
+++ b/src/main.py
@@ -1,9 +1,9 @@
 from os import getenv
 import discord

-from classes.challenge import Challenge
-from classes.user import User
-from pngmaker.new_validated_challenge import NewValidatedChallenge
+from src.classes.challenge import Challenge
+from src.classes.user import User
+from src.pngmaker.new_validated_challenge import NewValidatedChallenge


 SECRET = getenv('USELESS_SECRET')
diff --git a/src/pngmaker/new_validated_challenge.py b/src/pngmaker/new_validated_challenge.py
index 6a3a663..f6c47c3 100644
--- a/src/pngmaker/new_validated_challenge.py
+++ b/src/pngmaker/new_validated_challenge.py
@@ -1,7 +1,7 @@
 from os import path
 from PIL import Image,ImageDraw,ImageFont
-from classes.challenge import Challenge
-from classes.user import User
+from src.classes.challenge import Challenge
+from src.classes.user import User
 $ pylint --rcfile=.pylintrc $(git ls-files '*.py')

-------------------------------------------------------------------
Your code has been rated at 10.00/10
```
it solves the problem quite cleanly I think!

Reply:
Solved the pylint indeed but (on my side at least) the execution however does not work anymore : 
```
root@b0ea63dda5ba:/opt/root-pythia/src# python3 main.py
Traceback (most recent call last):
  File "/opt/root-pythia/src/main.py", line 4, in <module>
    from src.classes.challenge import Challenge
ModuleNotFoundError: No module named 'src'
```

Reply:
as discussed in private we can't find a really good solution here:
```
(basé sur 0a2d679 go back to realpath)
pytest ✅ ; python src/main.py ✅ ; pylint ❌
 
 Solution 1️⃣ 
pour pylint il faut passer le PYTHONPATH: PYTHONPATH=src pylint --rcfile=.pylintrc $(git ls-files '*.py')

 Solution 2️⃣ 
rm src/__init__.py et pylint --rcfile=.pylintrc $(git ls-files '*.py') ✅ --> tout marche meme si c'est un mystere pq pytest crash pas: l'import depuis tests/ de src jsp comment il passe j'avoue
```